core/vm/gas.go

```diff
@@ -24,6 +24,7 @@ import (
 const (
 	GasQuickStep   uint64 = 2
 	GasFastestStep uint64 = 3
+	GasFastishStep uint64 = 4
```

Comment:
Oy vey. This is a silly. Our enumeration from slower to faster goes: 
`Fast, fastish, fastest, and quick`. We have a superlative at penultimate position, but apparently quick beats fastest. 
Not your fault, though...